ext/cfx-ui/src/cfx/apps/mpMenu/services/intl/intl.mpMenu.ts

```diff
@@ -24,8 +24,17 @@ class MpMenuIntlService implements IIntlService {
     const [language, country] = systemLocale.split('-');
 
     if (!country) {
+      const lcLanguage = language.toLowerCase();
+      const ucLanguage = language.toUpperCase();
+
+      // Special case for Korean locale: On some systems, the Korean locale might be returned as `ko-KO`,
```

Comment:
My colleague pointed out that `systemLocale` may be just `ko` on some systems instead.

In this case the solution that you proposed works. We can keep it as it is. But lets update the comment to "the Korean locale might be returned as `ko`," to avoid future confusion.

Reply:
Should I make the edits and commit them, or will you do it?

Reply:
Please make and commit the change.

But, as mentioned earlier (https://github.com/citizenfx/fivem/pull/2745#issuecomment-2315573471) - to keep the history clean, you can just force update the current commit instead of creating the new one. In this case it would be
```
git commit --amend -m "new commit message"
git push --force
```

Also, while we are here, lets update the commit message to something like "fix(ext/cfx-ui): South Korea system locale", so it better corresponds to that changes that are made.